packages/design-system/src/components/web-components/ds-filter-chip/ds-filter-chip.tsx

```diff
@@ -0,0 +1,54 @@
+import { define } from '../preactement/define';
+import FilterChip from '../../FilterChip/FilterChip';
+import { FilterChipProps } from '../../FilterChip/FilterChip';
+import { parseBooleanAttr } from '../wrapperUtils';
+
+const attributes = [
+  'clear-label',
+  'class-name',
+  'label',
+  'root-id',
+  'size',
+  'use-alternate-icon',
+] as const;
+
+/* eslint-disable @typescript-eslint/no-namespace */
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'ds-filter-chip': React.DetailedHTMLProps<
+        React.HTMLAttributes<HTMLElement> & {
+          [K in (typeof attributes)[number]]?: string;
+        },
+        HTMLElement
+      >;
+    }
+  }
+}
+/* eslint-enable */
+
+interface WrapperProps extends Omit<FilterChipProps, 'useAlternateIcon'> {
+  useAlternateIcon?: string;
+  clearLabel?: string;
+}
+
+const Wrapper = ({ clearLabel, label, useAlternateIcon, ...otherProps }: WrapperProps) => (
+  <FilterChip
+    ariaClearLabel={clearLabel ?? ''}
```

Comment:
I'm curious about the semantics of passing an empty string here. Is there a specific accessibility concern we're addressing?

Reply:
So the `ariaClearLabel` prop/attribute isn't _actually_ an aria attribute. It's a string that gets passed into a span element. Because the attributes for web components have to be kebab case (aria-clear-label instead of ariaClearLabel), our a11y tests correctly point out that `aria-clear-label` isn't a real aria label. So I made up a new attribute for the web component that drops the `aria-` prefix and then maps onto the `ariaClearLabel` prop when we pass it into the Preact component. Since it's an optional prop and not written onto an actual `aria` attribute anywhere, I figured it would be ok to pass an empty string to it.